curvefs/src/client/s3/client_s3_cache_manager.cpp

```diff
@@ -275,67 +275,95 @@ int FileCacheManager::Read(uint64_t inodeId, uint64_t offset, uint64_t length,
         return readOffset;
     }
 
-    std::vector<S3ReadRequest> totalS3Requests;
-    auto iter = totalRequests.begin();
-    uint64_t fileLen;
     {
+        unsigned int maxRetry = 3;  // hardcode, fixme
```

Comment:
did you add test case or modify existing test case about this?

Reply:
maybe later